### PR TITLE
Updating results for Firefox 39 to Firefox 44

### PR DESCRIPTION
**Pull request by afmenez**
Updating results for Firefox 39 to Firefox 44.
A few corrections, no surprises.

### CODE REVIEW

File: data-es6.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ exports.browsers = {
   firefox37: {
     full: 'Firefox',
     short: 'FF 37',
-    obsolete: very,
```
Comment (ljharb) [inline data-es6.js:275]: why remove the trailing commas?

Comment (afmenez) [reply data-es6.js:275]: It was not on purpose, I removed these lines during my testing and copied back from FF36 when I finished.  But the comma is not necessary on the last item, and all other versions above this don't have it.

Comment (ljharb) [reply data-es6.js:275]: We should be consistent; and they should probably all have it to keep diffs small.

Comment (afmenez) [reply data-es6.js:275]: Done.

**Review by ljharb — APPROVED** [review]
Thanks, LGTM

**Pull request merged**